helm/thingsboard/templates/initializedb-job.yaml

```diff
@@ -44,11 +44,16 @@ spec:
           - mountPath: /config
             name: '{{ .Release.Name }}-node-config'
         image: "{{ .Values.node.image.server | default .Values.global.image.server }}/{{ .Values.node.image.repository }}:{{ .Values.node.image.tag | default .Values.global.image.tag }}"
-        command:
-          - "/bin/bash"
+        command: ["/bin/bash", "-c"]
         args:
-          - "-c"
-          - "start-tb-node.sh"
+          -|
+          start-tb-node.sh; \
+          export PGPASSWORD="{{ index .Values "postgresql-ha" "postgresql" "password" }}" \
+          psql -w -h "{{ include "thingsboard.pgpoolservicename" . }}" \
```

Comment:
looking at init-container examples, it might require `--set=on_error_stop=1`

Reply:
actually to avoid this restarting the full tb init, it might be worth moving to another container that gets signaled to start by this container touching a file in a common mounted volume.